cmd/relayproxy/config/retriever.go

```diff
@@ -13,25 +13,27 @@ type RetrieverConf struct {
-	URL          string              `mapstructure:"url" koanf:"url"`
-	Timeout      int64               `mapstructure:"timeout" koanf:"timeout"`
-	HTTPMethod   string              `mapstructure:"method" koanf:"method"`
-	HTTPBody     string              `mapstructure:"body" koanf:"body"`
-	HTTPHeaders  map[string][]string `mapstructure:"headers" koanf:"headers"`
-	Bucket       string              `mapstructure:"bucket" koanf:"bucket"`
-	Object       string              `mapstructure:"object" koanf:"object"`
-	Item         string              `mapstructure:"item" koanf:"item"`
-	Namespace    string              `mapstructure:"namespace" koanf:"namespace"`
-	ConfigMap    string              `mapstructure:"configmap" koanf:"configmap"`
-	Key          string              `mapstructure:"key" koanf:"key"`
-	BaseURL      string              `mapstructure:"baseUrl" koanf:"baseurl"`
-	AuthToken    string              `mapstructure:"token" koanf:"token"`
-	URI          string              `mapstructure:"uri" koanf:"uri"`
-	Database     string              `mapstructure:"database" koanf:"database"`
-	Collection   string              `mapstructure:"collection" koanf:"collection"`
-	RedisOptions *redis.Options      `mapstructure:"redisOptions" koanf:"redisOptions"`
-	RedisPrefix  string              `mapstructure:"redisPrefix" koanf:"redisPrefix"`
+	GithubToken    string              `mapstructure:"githubToken" koanf:"githubtoken"`
+	URL            string              `mapstructure:"url" koanf:"url"`
+	Timeout        int64               `mapstructure:"timeout" koanf:"timeout"`
+	HTTPMethod     string              `mapstructure:"method" koanf:"method"`
+	HTTPBody       string              `mapstructure:"body" koanf:"body"`
+	HTTPHeaders    map[string][]string `mapstructure:"headers" koanf:"headers"`
+	Bucket         string              `mapstructure:"bucket" koanf:"bucket"`
+	Object         string              `mapstructure:"object" koanf:"object"`
+	Item           string              `mapstructure:"item" koanf:"item"`
+	Namespace      string              `mapstructure:"namespace" koanf:"namespace"`
+	ConfigMap      string              `mapstructure:"configmap" koanf:"configmap"`
+	Key            string              `mapstructure:"key" koanf:"key"`
+	BaseURL        string              `mapstructure:"baseUrl" koanf:"baseurl"`
+	AuthToken      string              `mapstructure:"token" koanf:"token"`
+	URI            string              `mapstructure:"uri" koanf:"uri"`
+	Database       string              `mapstructure:"database" koanf:"database"`
+	Collection     string              `mapstructure:"collection" koanf:"collection"`
+	RedisOptions   *redis.Options      `mapstructure:"redisOptions" koanf:"redisOptions"`
+	RedisPrefix    string              `mapstructure:"redisPrefix" koanf:"redisPrefix"`
+	Workspace      string              `mapstructure:"bitbucketWorkspace" koanf:"bitbucketWorkspace"`
```

Comment:
**suggestion:** ✂️ After applying this [suggestion](https://github.com/thomaspoignant/go-feature-flag/pull/2611/files#r1828203189), we can remove this configuration from the relay proxy.